### DISCUSSION TITLE
Seasons - an idea on how they could work in osu!lazer

### DISCUSSION THREAD

**Discussion by enenbon** [ppy/osu#13066] [Ideas]
### **What are "Seasons"?**
Seasons are a new, and fun way for players to compete against eachother. They will introduce rotating mapsets with seasonal leaderboards, which will be separate from the maps' regular leaderboard.

### **How would Seasonal Leaderboards work?**
Every 3 months, a selection of maps will be put into a "Season". They will have new, seasonal leaderboards, which will be separate from the regular ones. There could also be brackets for these seasonal leaderboards, to give all players a fighting chance. Once the 3 months are over and the season rotates, new maps get selected, with new seasonal leaderboards, and the cycle continues.

### **What's the point? Aren't the leaderboards we have now good enough?**
The leaderboards we have right now are great, but with no rank brackets, or mod restrictions, lower-skilled players don't have a chance even on the difficulty that they're comfortable on. With seasons, there could be mod restrictions and rank brackets, so that, for example, people between ranks 25,000-50,000 can compete against eachother on the same maps without worrying that a top 100 player will take over the leaderboard. With this, it also gives higher ranked players the opportunity to compete against eachother on hard maps, to see who can end the season on rank 1 on a said map. In a way, seasons can be looked at as extremely large scale tournaments. Players could also receive a profile badge if they top a leaderboard on a map when the season ends. 

I hope this is an interesting enough of an idea for it to atleast be considered in some way. Of course, all the details aren't precise here - the length of the season, the number of maps per season, the rank brackets, the mod restrictions per season, etc, have to all be decided, but this is my interpretation of how it could work.
Bear in mind, I am no programmer, so I don't know the technicalities behind how this could work in lazer, if it even can.

**Comment by peppy** [comment]
This is planned, and kind of already exists in [spotlights](https://osu.ppy.sh/rankings/osu/charts).
